LEAF_Request_Portal/templates/subindicators.tpl

```diff
@@ -858,8 +858,8 @@
                 }
             };
             <!--{/if}-->
-            if(grpSel != undefined) {
-                var grpSel = {};
+            if(typeof leaf_groupSelector == 'undefined') {
+                var leaf_groupSelector = {};
```

Comment:
Did we want lets here? Don't know if this could cause issues in scoping

Reply:
The scoping rule was a bit confusing, and I'm not sure of the best way to express this. Since this block could potentially be included in the DOM many times, "let" would flag a variable redeclaration error.

The intent here is to make leaf_groupSelector available as a global variable. And when the block is included again, it shouldn't wipe the contents of the variable.

Would this syntax make sense? (Just committed in this PR as [6a88fb5](https://github.com/department-of-veterans-affairs/LEAF/pull/1713/commits/6a88fb5d2874afb0fb645f1c532b40de0ccd0c79))
```js
var leaf_groupSelector = leaf_groupSelector ?? {};
```

Reply:
Yes this would work better.